.github/workflows/prerelease.yml

```diff
@@ -167,3 +167,15 @@ jobs:
         env:
           SLACK_WEBHOOK_URL: "${{ secrets.SLACK_WEBHOOK_URL }}"
           SLACK_WEBHOOK_TYPE: "INCOMING_WEBHOOK"
+
+  deploy-sandbox:
+    if: startsWith(github.event.release.tag_name, 'v2.')
```

Comment:
Github actions doesn't give us the branch information when the workflow is initiated by a release so I think this is the best we can do.